src/EdFi.AnalyticsMiddleTier.DataStandard32/Chrab/PostgreSQL/0003-View-ChronicAbsenteeismAttendanceFact-Update.sql

```diff
@@ -3,6 +3,8 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
+DROP VIEW IF EXISTS analytics.chrab_ChronicAbsenteeismAttendanceFact;
+
 CREATE OR REPLACE VIEW analytics.chrab_ChronicAbsenteeismAttendanceFact
```

Comment:
Why would you need to add a `DROP VIEW` statement when line 6 (now 8) already has `OR REPLACE` in it?

Reply:
That surprised me too. I took this from the postgres documentation:
<img width="819" alt="Capture" src="https://user-images.githubusercontent.com/16614428/103695064-89f6b180-4f61-11eb-8a38-f96a10961f68.PNG">

Here is the link: https://www.postgresql.org/docs/current/sql-createview.html

Reply:
How interesting and unexpected. I guess that means the columns had changed previously. OK, thanks for the info.